### PR TITLE
thc-pptp-bruter: use https, modernized

### PR DESCRIPTION
**Pull request by vszakats**
thc-pptp-bruter: use https, modernized

### CODE REVIEW

File: Library/Formula/thc-pptp-bruter.rb [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
 
   def install
     # The function openpty() is defined in pty.h on Linux, but in util.h on OS X.
-    # See http://groups.google.com/group/sage-devel/msg/97916255b631e3e5
-    inreplace 'src/pptp_bruter.c', 'pty.h', 'util.h'
+    # See https://groups.google.com/group/sage-devel/msg/97916255b631e3e5
+    inreplace "src/pptp_bruter.c", "pty.h", "util.h"
 
     system "./configure", "--prefix=#{prefix}"
     system "make"
```
Comment (mikemcquaid) [inline Library/Formula/thc-pptp-bruter.rb]: If we're modernising can you run `brew audit --strict` and fix the line below and add a test too? Thanks!

Comment (vszakats) [inline Library/Formula/thc-pptp-bruter.rb]: For a test I'd need to know what this package does, which I don't, I'm sorry.

As for the `brew audit`, I'm still wrestling with an error:
```
Error: The 'rubocop' gem is installed but couldn't find 'rubocop' in the PATH:
```
The PATH added by rubygems is 2.0.0 (system ruby), while I'm using 2.2.0 from Homebrew. Still in search for the solution...

Comment (vszakats) [inline Library/Formula/thc-pptp-bruter.rb]: Band-aided by adding proper PATH manually. It's working fine now.

Comment (mikemcquaid) [inline Library/Formula/thc-pptp-bruter.rb]: Ok, cool. For a test it's worth trying to just run an executable provided with the software as a `system` call.

**Comment by vszakats** [comment]
Will reopen this one, messed up the rebase/squash. (Learning a lot)

**Pull request closed without merging**